app/models/bgs_dependents/child_student.rb

```diff
@@ -7,8 +7,7 @@ class ChildStudent < Base
     attribute :student_networth_information, Hash
     attribute :student_expected_earnings_next_year, Hash
 
-    def initialize(dependents_application, proc_participant)
```

Comment:
Here also (same as above)

Reply:
(Same as above) We've been doing a lot of refactoring due to breaking up a larger PR into multiple smaller PRs. This functionality has been moved to /lib/bgs/student_school.rb in the submit_params method.

Reply:
Not to make it even more confusing, but due to some more refactoring and feedback suggestions, we've added this back in the params_for_686c method.